Lib/test/test_posix.py

```diff
@@ -647,6 +647,33 @@ def test_stat(self):
                 'should be string, bytes, os.PathLike or integer, not',
                 posix.stat, list(os.fsencode(os_helper.TESTFN)))
 
+        # see gh-102890
+        code = f"""if 1:
```

Comment:
This should be a separate test case, please add a new `test_` method.

Reply:
Done.